services/clusters-keeper/src/simcore_service_clusters_keeper/rpc/clusters.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import FastAPI
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from servicelib.rabbitmq import RPCRouter
+
+from ..core.errors import Ec2InstanceNotFoundError
+from ..core.settings import get_application_settings
+from ..models import ClusterGet, EC2InstanceData
+from ..modules import clusters
+from ..modules.dask import ping_gateway
+from ..utils.dask import get_gateway_url
+
+router = RPCRouter()
+
+
+@router.expose()
+async def get_or_create_cluster(
+    app: FastAPI, *, user_id: UserID, wallet_id: WalletID
+) -> ClusterGet:
```

Comment:
Q: would it make sense to put `ClusterGet` in `models_library` so that the client can reuse it?

Reply:
this will move in the next PR already,.